sycl/test/regression/host_tanpi_double_accuracy.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %t.out
+//
+// Checks that sycl::tanpi with double has the accuracy required by the
+// specification. Reference results are taken from SYCL-CTS.
+
+#include <cmath>
+#include <iomanip>
+#include <sycl/sycl.hpp>
+
+double get_ulp(double X) {
+  double Inf = std::numeric_limits<double>::infinity();
+  double Negative = std::fabs(std::nextafter(X, -Inf) - X);
+  double Positive = std::fabs(std::nextafter(X, Inf) - X);
+  return std::fmin(Negative, Positive);
+}
+
+int check_tanpi(double input, double ReferenceRes, unsigned int Acc = 6) {
```

Comment:
I guess the default value for `Acc` comes from SYCL2020 requirements. Can you leave a comment explaining it?

Reply:
I have added a comment above. Does this sufficiently clarify why `Acc` is 6?

Reply:
That's perfect!